test/Test/Oscoin/Data/OscoinTx.hs

```diff
@@ -1,30 +1,25 @@
-module Test.Oscoin.Data.OscoinTx where
+module Test.Oscoin.Data.OscoinTx (tests, genTx) where
```

Comment:
I am grateful for the `genTx` generator -- it will make my life of integrating the last part of the Sync tests with this new `SQLite` store much easier 😉

Reply:
I am grateful for having been able to write `arbitrary = hedgehog genTx` :dancer: